client/packages/electron/src/electron.ts

```diff
@@ -110,7 +109,6 @@ class Scanner {
         }
       }
     });
-    return devices;
   }
```

Comment:
`startDeviceScan: () => Promise<BarcodeScanner[]>` <- startDeviceSacn is meant to return devices, I think we don't actually use the return, but the NativeApi contract seems to suggest we do, I think worth changing the contract (NativeApi type or continue returning devices)